src/qt/pivx/send.cpp

```diff
@@ -221,18 +221,21 @@ void SendWidget::loadWalletModel() {
 }
 
 void SendWidget::clearAll(){
-    onResetCustomOptions();
+    onResetCustomOptions(false);
     if(customFeeDialog) customFeeDialog->clear();
     ui->pushButtonFee->setText(tr("Customize Fee"));
     if(walletModel) walletModel->setWalletDefaultFee();
     clearEntries();
     refreshAmounts();
 }
 
-void SendWidget::onResetCustomOptions(){
+void SendWidget::onResetCustomOptions(bool fRefreshAmounts){
     CoinControlDialog::coinControl->SetNull();
     ui->btnChangeAddress->setActive(false);
```

Comment:
I suggest to add something like this here
```cpp
coinControlDialog = nullptr;
```

Otherwise, even when the coinControl is set to null, the rows remain selected in the coinControl dialog (reopening it after clicking "Reset to default").